app/plugin/jira/webhook.go

```diff
@@ -92,7 +92,11 @@ func (w *Webhook) SlackAttachment() (*model.SlackAttachment, error) {
 	text, err := w.renderText("" +
 		"[{{.Issue.Fields.Summary}}]({{.JIRAURL}}/browse/{{.Issue.Key}})" +
 		"{{if eq .WebhookEvent \"jira:issue_created\"}}{{if ne .Issue.Fields.Description \"\"}}" +
+		"{{if len .Issue.Fields.Description | lt 3000}}" +
```

Comment:
Pipelines pass their input as the _last_ argument in the next function. It's been bothering me too how this reads unintuitively.